datamodel/extra_scripts/oracle/208_bag2_views.sql

```diff
@@ -219,10 +219,10 @@ from (select 'true'
              a.huisletter,
              a.huisnummertoevoeging,
              a.postcode,
-             (select listagg(vbod.gebruiksdoel, ', ')
-              from verblijfsobject_gebruiksdoel vbod
-              where (vbod.identificatie = voa.identificatie and vbod.voorkomenidentificatie =
-                                                               voa.voorkomenidentificatie))                             as maaktdeeluitvan,
+             (select listagg(vbod.maaktdeeluitvan, ', ')
+              from verblijfsobject_maaktdeeluitvan vbod
+              where (vbod.identificatie = vo.identificatie and vbod.voorkomenidentificatie =
```

Comment:
deze is de foutmelding:
```
SQL> SQL> SQL> SQL>   2    3    4    5    6    7    8    9   10   11   12   13   14   15   16   17   18   19   20   21   22   23   24   25   26   27   28   29   30   31   32   33   34   35   36   37   38   39   40   41   42   43   44   45   46   47   48   49   50   51   52   53   54   55   56   57   58   59   60   61   62   63   64   65   66   67   68   69   70   71   72   73   74  							     vo.voorkomenidentificatie))			     as maaktdeeluitvan,
                                                             *
ERROR at line 63:
ORA-00904: "VO"."VOORKOMENIDENTIFICATIE": invalid identifier

```

De alias `vo` is niet gedefinieerd, wel `voa`

Reply:
ik zie het ja!